libs/core/src/components.d.ts

```diff
@@ -2064,6 +2116,51 @@ declare namespace LocalJSX {
          */
         "variant": 'primary' | 'availability' | 'filter';
     }
+    interface PdsText {
+        /**
+          * Sets the text alignment.
+         */
+        "align"?: 'start' | 'center' | 'end' | 'justify';
+        /**
+          * Sets the text color.
+         */
+        "color"?: | 'primary'
+  | 'secondary'
+  | 'neutral'
+  | 'accent'
+  | 'danger'
+  | 'info'
+  | 'success'
+  | 'warning';
+        /**
+          * Sets the font size.
+         */
+        "size"?: | '2xl'
+  | 'xl'
+  | 'lg'
+  | 'md'
+  | 'sm'
+  | 'xs'
+  | '2xs';
+        /**
+          * Determines what semantic text tag to render.
+         */
+        "tag": | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | 'h6'
+  | 'p'
+  | 'code'
+  | 'pre'
+  | 'strong'
+  | 'em';
+        /**
+          * Sets the font weight.
+         */
+        "weight"?: 'regular' | 'medium' | 'semibold' | 'bold';
+    }
```

Comment:
Can truncate be a prop that displays the attribute?

Reply:
@QuintonJason Per discussion, truncate is just an attribute added to the component that only controls styling.